packages/opentelemetry/src/spanProcessor.ts

```diff
@@ -46,20 +45,35 @@ function onSpanStart(span: Span, parentContext: Context): void {
   client?.emit('spanStart', span);
 }
 
-function onSpanEnd(span: Span): void {
+function onSpanEnd(span: ReadableSpan): void {
   const client = getClient();
-  client?.emit('spanEnd', span);
+  client?.emit('spanEnd', span as Span);
```

Comment:
m: Is this safe? If so, why?

Reply:
I do realize that it was like this before, but I now wonder whether we can depend on this. But nothing to duke out in this pr!

Reply:
I checked the actual implementation in the tracer, and I think it's fine - not sure how we could be even safer here without checking the existence of each and every property we'd expect this to have 😬 which also feels overkill. But we can for sure revisit this!